.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 dist
 .DS_Store
 *.log
-package-lock.json
\ No newline at end of file
+package-lock.json
+.idea
```

Comment:
Shouldn't we keep `.gitignore` for project-related files/directories only and move others (ide-related, system-related..) to our own global gitignore file?

Reply:
I did not know about global gitignore. Thanks for feedback.